R/class-workbook.R

```diff
@@ -5516,36 +5442,33 @@ wbWorkbook <- R6::R6Class(
           values
         ),
 
-        ## between ----
         between = sprintf(
           '<cfRule type="cellIs" dxfId="%s" priority="1" operator="between"><formula>%s</formula><formula>%s</formula></cfRule>',
           dxfId,
           formula[1],
           formula[2]
         ),
 
-        ## topN ----
         topN = sprintf(
           '<cfRule type="top10" dxfId="%s" priority="1" rank="%s" percent="%s"></cfRule>',
           dxfId,
-          values[1],
-          values[2]
+          values$rank,
+          if (values$percent == 1) 1 else "NULL"
```

Comment:
not sure if null is a valid option here. would have expected `0`

Reply:
`0` seems better.  Our tests were expecting `"NULL"` but maybe that wasn't the best.